src/common/services/registerAccount.service.js

```diff
@@ -0,0 +1,86 @@
+import angular from 'angular';
+import orderService from 'common/services/api/order.service';
+import sessionModalService from 'common/services/session/sessionModal.service';
+import sessionService from 'common/services/session/session.service';
+
+import {Roles} from 'common/services/session/session.service';
+
+let serviceName = 'registerAccountService';
+
+/*@ngInject*/
+function RegisterAccountService( $q, orderService, sessionService, sessionModalService ) {
+
+// angular.element(document.querySelector('body')).injector().get('registerAccountService')().then(function() {console.log('success');}, function() {console.log('failure')});
```

Comment:
I just had it in there so I'd remember it, since chrome incognito doesn't keep it in history. I'm going to remove it before final PR.